lib/views/home/home_screen.dart

```diff
@@ -5,7 +5,7 @@ import 'package:thc/models/user.dart';
 import 'package:thc/views/admin_portal/admin_portal.dart';
 import 'package:thc/views/widgets.dart';
 
-class HomeScreen extends StatelessWidget {
+class HomeScreen extends StatefulWidget {
```

Comment:
```suggestion
class HomeScreen extends StatelessWidget {
```

This widget should still be stateless. (One way to tell them apart: stateless widgets have a `build()` method, and stateful widgets have a `createState()` method.)

Reply:
Thanks! Just fixed that!